cypress/support/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const projectUrl = Cypress.env("BASE_URL");
+export const stellarIconUrl = "https://stellar.org/";
```

Comment:
This is not an icon but a URL
```suggestion
export const stellarUrl = "https://stellar.org/";
```

Reply:
Solved in commit 7efb5acd746344ce0dd24e6411532b17f3975581.